test/client/spec/victory-primitives/candle.spec.js

```diff
@@ -39,7 +34,6 @@ describe("victory-primitives/candle", () => {
     expect(rect.prop("height")).to.eql(20);
     // x = x - width / 2
     expect(rect.prop("x")).to.eql(4);
-    expect(rect.prop("y")).to.eql(30);
```

Comment:
Shouldn't `expect(rect.prop("y")).to.eql(30);`still be true?

Reply:
I would have thought so, but it returns `undefined` when I run the test

Reply:
Tooling around with it more and I've graduated from `undefined` to `NaN`.

Reply:
Looks like you need to add `close` and `open` to the props you're using to render the test component.